tests/testdata/passwords

```diff
@@ -12,3 +12,4 @@ disableduser:DgT1djMaFfpEs:1112:1112:::
 lockeduser:DgT1djMaFfpEs:1113:1113:::
 usernotoken::1114:1114:::
 multichal::1115:1115:::
+nönäscii:IXsH4ttQlw2xA:1116:1116:Nön,field2,field3,field4,field5::
```

Comment:
You know you are hardcore, since the original password file does not support special characters. ;-)

Reply:
Hah yeah, I just remembered we talked about adding unicode support to the passwd id resolver in #754, so I figured why not just do it now :)